client/src/App.js

```diff
@@ -3,16 +3,18 @@ import AppHeader from "./components/AppHeader/AppHeader";
 import GlobalStyle from "./GlobalStyle";
 import Button from "./components/Button/Button";
 import React from "react";
+import Map from "./components/Map/Map";
 
 function App() {
   return (
     <div className="App">
-      <GlobalStyle />
+      <Map />
+      {/* <GlobalStyle />
```

Comment:
I know that this is a draft request, but consider removing these lines instead of comment them out to avoid having unused code.